src/Entity/MemberSummary/JobExperience.php

```diff
@@ -128,6 +128,11 @@ class JobExperience implements SummaryItemPositionableInterface
      */
     private $summary;
 
+    public function __toString(): string
+    {
+        return $this->company;
```

Comment:
You use `experiences.company` in the filtre, you don't really need this, do you ?

Reply:
totally.
it was useful before we decided to split experiences.company and experiences.name